lamindb/models/artifact.py

```diff
@@ -1662,12 +1668,13 @@ def from_mudata(
             :class:`~lamindb.Feature`
                 Track features.
 
-        Examples:
-            >>> import bionty as bt
-            >>> bt.settings.organism = "human"
-            >>> mdata = ln.core.datasets.mudata_papalexi21_subset()
-            >>> artifact = ln.Artifact.from_mudata(mdata, description="a mudata object")
-            >>> artifact.save()
+        Example::
+
+            import lamindb as ln
+
+            mdata = ln.core.datasets.mudata_papalexi21_subset()
+            artifact = ln.Artifact.from_mudata(mdata, key="mudata_papalexi21_subset.h5mu")
+            artifact.save()
```

Comment:
move save() on the previous line

Reply:
I thought of it, but think this is better, because otherwise if user run into errors, it's hard to understand where it comes from.